zc_plugins/SystemInspection/1.6.0/admin/includes/extra_configures/system_inspection_base.php

```diff
@@ -319,6 +319,7 @@
 
 define('OBSOLETE_SHIPPING', [
     'ups.php',
+    'usps.php',
```

Comment:
Shouldn't 'fedex.php' be in that list, too?

Reply:
Agreed. Added